packages/semver/src/executors/version/utils/try-bump.ts

```diff
@@ -57,21 +69,50 @@ If your project is already versioned, please tag the latest release commit with
     )
   );
 
+  let isRetry = false;
```

Comment:
Semver is tag based to find the last version, adding a retry logic if the tag is not found could lead to undesired behaviors. It's something we could discuss but it's not related to the subject of this PR. I think we should drop this for now.

Reply:
I thought there were some certain cases where the retry logic was required, but tried running tests without the retry logic and found all tests pass. It's been a month since this PR was created so please understand :smiling_face_with_tear: